local/include/etc/confluent/docker/kafka-tools-log4j.properties.template

```diff
@@ -0,0 +1,6 @@
+log4j.rootLogger={{ getEnv "KAFKA_TOOLS_LOG4J_LOGLEVEL" "WARN" }}, stderr
```

Comment:
is this file any different than the version in cp-kafka?

Reply:
Yes. This template file is compatible with golang to generate the properties file.
In cp-kafka, the template file is for python. So syntaxes are different.